lib/ansible/modules/system/filesystem.py

```diff
@@ -45,14 +46,15 @@
     choices: [ "yes", "no" ]
     default: "no"
     description:
-    - If yes, allows to create new filesystem on devices that already has filesystem.
+    - If C(yes), allows to create new filesystem on devices that already has filesystem.
     required: false
   resizefs:
     choices: [ "yes", "no" ]
     default: "no"
     description:
-    - If yes, if the block device and filesytem size differ, grow the filesystem into the space. Note, XFS Will only grow if mounted. btrfs and reiserfs
-    don't support resizing.
+    - If C(yes), if the block device and filesytem size differ, grow the filesystem into the space.
+    - XFS Will only grow if mounted.
+    - C(btrfs) and C(reiserfs) don't support resizing.
```

Comment:
It might be better to say "all other filesystems do not support resizing". (so adding a filesystem mean that we have one less place to modify)